packages/element/src/react-platform.js

```diff
@@ -43,6 +44,20 @@ export { render };
  */
```

Comment:
Should we add a `@deprecated` for `render()` and `hydrate()` now that they are deprecated in React 18?

Reply:
I think we if add @deprecated, we need to also add a real deprecated function call, so let's leave it for later when we migrate all of our usage.

Reply:
Sure. Maybe adding a comment would suffice then.